actions/lib/utils.py

```diff
@@ -0,0 +1,33 @@
+import re
+from github.InputGitAuthor import InputGitAuthor
+from github.GithubObject import NotSet
+
+__all__ = [
+    'prep_github_params_for_file_ops'
+]
+
+
+def create_github_author(s):
+    # expecting string in the format of "FirstName LastName <email@address>"
+    pattern = re.compile(r"^(.*?)\s+<([a-zA-Z0-9_.+-]+?@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+)?>$")
```

Comment:
for performance reasons, probably good to compile this at the file level and the reuse the compiled pattern within this function

Reply:
OK.